chainercv/links/model/resnet/resnet.py

```diff
@@ -0,0 +1,281 @@
+from __future__ import division
+
+import numpy as np
+
+import chainer
+import chainer.functions as F
+from chainer import initializers
+import chainer.links as L
+
+from chainercv.links import Conv2DBNActiv
+from chainercv.links.model.resnet.resblock import ResBlock
+from chainercv.links import PickableSequentialChain
+from chainercv.utils import download_model
+
+
+# RGB order
+# This is channel wise mean of mean image distributed at
+# https://github.com/KaimingHe/deep-residual-networks
+_imagenet_mean = np.array(
+    [123.15163084, 115.90288257, 103.0626238],
+    dtype=np.float32)[:, np.newaxis, np.newaxis]
+
+
+class ResNet(PickableSequentialChain):
+
+    """Base class for ResNet Network.
+
+    This is a feature extraction link.
```

Comment:
`pickable sequential link`? We have not defined `feature extraction link` officially.